documentation/data-importer.adoc

```diff
@@ -138,6 +136,28 @@ If you're unsure if you've followed the steps correctly so far, you can go ahead
 button::Load Northwind node mapping[role=NX_IMPORT_LOAD,endpoint={northwind-subset-node-only-mapping-zip-file}]
 ====
 
+== Constraints and indexes
+
+To ensure accuracy of your data and support read performance, Data Importer automatically creates a uniqueness constraint and a corresponding index on the property you select as the node ID.
+You can't modify the two that are created for you, but you can add additional indexes.
+
+Constraints and indexes have their own tab in the highlight:import/import-mapping-panel[mapping details panel].
+Use the `+`-sign to add an index and then select which property to add the index on.
+Only the properties you mapped from the _Definition_ tab are available.
+Data Importer generates a name for the index and the index type created is Neo4j's default index.
+If you make a mistake, select the erroneous index and click the trash-can to delete it.
+
+If you know that you are going to use a property regularly, it is good practice to add an index to it.
+For example, if you doing administrative work with Northwind, you may be interested in the order dates.
+See if you can add an index that will help you that!
```

Comment:
I would add a little more scaffold to this, maybe 

See if you can add an index to the `x` property of the `x` nodes to help with that!

[I dont know what the property and node would be in the northwind database]